perf/benchmark/runner/runner.py

```diff
@@ -85,17 +90,14 @@ def __init__(
             clientsidecar=False,
             bothsidecar=True,
             ingress=None,
-            mesh="istio",
-            cacert=None):
+            mesh="istio"):
         self.run_id = str(uuid.uuid4()).partition('-')[0]
         self.conn = conn
         self.qps = qps
         self.size = size
         self.duration = duration
         self.mode = mode
-        self.ns = os.environ.get("NAMESPACE", "twopods")
-        # bucket resolution in seconds
-        self.r = "0.00005"
```

Comment:
Resolution is removed? Is it non modifiable?

Reply:
Today histogram resolution is fixed in NH. For double checking if the current `HdrHistogram` set up suffices, please see [here](https://github.com/envoyproxy/nighthawk/blob/master/source/common/statistic_impl.cc#L162) and the tests with respect to precision over [here](https://github.com/envoyproxy/nighthawk/blob/master/test/statistic_test.cc#L49)

Reply:
4 significant digits, tracked from 1ns to a large number, it willl suffice.